ibm/mas_devops/roles/suite_uninstall/defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 ---
 mas_instance_id: "{{ lookup('env', 'MAS_INSTANCE_ID') }}"
+mas_wipe_mongo_data: "{{ lookup('env', 'MAS_WIPE_MONGO_DATA') | default(false) | bool }}"
```

Comment:
This should be:

"{{ lookup('env', 'MAS_WIPE_MONGO_DATA') | default('false', True) | bool }}"

You have to pass True in the second param, otherwise default teats empty string as if a value has been set instead of treating it as "I need to use the default".

Reply:
Good catch, forgot about it. Will fix it.